app.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  # List all books.
+  def list_books
+    puts 'List of Books:'
+    @books.each do |book|
+      puts "📚 Title: #{book.title} by #{book.author}"
+    end
+  end
+
+  # List all people.
+  def list_people
+    @persons.each do |person|
+      if person.instance_of?(Student)
+        puts "[Student 🧍], ID: #{person.id}, Name: #{person.name}, age: #{person.age}"
+      else
+        puts "[Teacher 👨‍🏫], ID: #{person.id}, Name: #{person.name}, age: #{person.age}"
+      end
+    end
+  end
+
+  # Create a person (teacher or student)
+  def create_a_person
+    puts 'Do you want to add a student (1) or a teacher (2)? [Insert the number]'
+    is_student = gets.chomp.to_i
+    puts 'Age'
+    age = gets.chomp.to_i
+    puts 'Name'
+    person_name = gets.chomp
+
+    case is_student
+    when 1
+      puts 'Has parent permission? [Y/N]'
+      gets.chomp.downcase
+      student = Student.new(age: age, name: person_name)
+      @persons.push(student)
+    when 2
+      puts 'Specialization'
+      specialization = gets.chomp
+      teacher = Teacher.new(age: age, specialization: specialization, name: person_name)
+      @persons.push(teacher)
+    end
+    puts '🎉 Person added successfully.'
+  end
+end
+
+public
+
+# Create a book
+def create_a_book
+  puts 'Title'
+  book_title = gets.chomp
+  puts 'Author'
+  book_author = gets.chomp
+  book = Book.new(title: book_title, author: book_author)
+  @books.push(book)
+  puts '🎉 Book added successfully.'
+end
+
+# Create a rental
+def create_a_rental
+  puts 'Select a book from the following list by number'
+  @books.each_with_index do |book, index|
+    puts "#{index}) Title: #{book.title}, Author: #{book.author}"
+  end
```

Comment:
![image](https://github.com/mohashyne/school_library/assets/93039359/d7de30e6-fb6e-44d3-a6ce-a76c6ed9076c)

- [ ] It would be nice if you make your console app behave the same way as in the given design by microverse 👍 because I can see when we render the books it shows us an object and in the same line we see no author 😨 which is alarming because it can create confusion for the user and is also not following the given design 🤔 Kindly make your console app return the books in the same format which is shown in the microverse console app 👍